hub/core/transform/transform_tensor.py

```diff
@@ -82,3 +81,6 @@ def extend(self, items):
         """Adds multiple items to the tensor."""
         for item in items:
             self.append(item)
+
+    def __del__(self):
+        self.items.clear()
```

Comment:
I am skeptical of this, as deletion of a view tensor can clear the items for base tensor. Maybe we should pass `item=self.items[:]` in `__getitem__` to avoid this.